playbooks/deploy-rock.yml

```diff
@@ -619,6 +639,7 @@
       repo: "{{ bro_rockscripts_repo }}"
       dest: /opt/bro/share/bro/site/scripts/rock
       version: "{{ bro_rockscripts_branch }}"
+      #update: no
```

Comment:
Good point. We could put that stuff in a distinct file generated by ansible and include it. 🤷‍